handbook/upgrade/upgrade.md

```diff
@@ -0,0 +1,280 @@
+
+1. Provision a new Windows server for SystemLink.
+
+1. Provision a PostgreSQL server or replica set.
+
+1. Install and configure the new version of SystemLink Server that supports PostgreSQL.
+
+    !!!important
+        If you are using the single node configuration for SystemLink, the migration from the local instances of MongoDB to PostgreSQL begins automatically. If you are migrating a multi node configuration where the original SystemLink Server used a dedicated MongoDB server or replica set, you must specify the PostgreSQL instance for SystemLink to use before the MongoDB to PostgreSQL migration can occur. The next step in this workflow assumes you are in this multi node configuration.
+
+1. Configure SystemLink to use the newly created PostgreSQL server or replica set and click **Apply**. Refer to [Connecting to a Remote PostgreSQL Database](https://www.ni.com/documentation/en/systemlink/latest/setup/remote-postgres-databse/) for this step.
+
+1. Install `nislmigrate` on your new SystemLink Server.
+
+1. Attach the `D:\` volume used to capture data from your original SystemLink server.
+
+1. Run the command `nislmigrate restore --all --secret <your secret> --dir D:\migration`
+
+    !!!note
+        After this step SystemLink will migrate your test steps, results, and products from MongoDB to PostgreSQL. Depending on the size of your data set this process may take some time.
+
+1. Verify your new SystemLink Server has all the expected migrated data.
+
+## Upgrading Multi Node Configurations
+
+The following workflow describes upgrading a SystemLink Server instance that has been configured to use a dedicated MongoDB server or replica set, a dedicated PostgreSQL server or replica set, and a dedicated file store such as AWS S3.
+
+1. Backup your SystemLink Server, MongoDB server or replica set, PostgreSQL server or replica set, and file store.
+
+    !!!note
+        If S3 is used as your file store the backup step is not needed since this managed internally by AWS.
+
+1. Attach a volume to store data captured by `nislmigrate`. This will be referenced as `D:\`.
+
+1. Install `nislmigrate`.
+
+1. Run the command `nislmigrate capture --tags --dir D:\migration`.
+
+    !!!note
+        While most of the data is external to SystemLink, tag current values are stored on the app server. Therefore this `nislmigrate` argument flag must be used to ensure that data is not lost during the migration.
+
+1. Detach `D:\`.
+
+    !!!note
+        At this point in the workflow your original SystemLink Server is in an operable state. Be aware if any new data is created or ingested by the SystemLink server at this time it will not be available to your new server.
+
+1. Provision a new Windows server for SystemLink.
+
+1. Provision a MongoDB server or replica set from the backup previously created.
+
+1. Provision a PostGres server or replica set from the backup previously created.
```

Comment:
Not sure if the correct capitalization is `PostGres` or `Postgres`. Might be worthwhile to just use the full `PostgreSQL` here to avoid  confusion.

Reply:
Good catch! I thought I had found all instances where I was inconsistent.